core/src/main/resources/hudson/PluginManager/_installed.js

```diff
@@ -0,0 +1,55 @@
+(function () {
+  const restartRequired = document
+    .querySelector("#is-restart-required-for-completion")
+    .getAttribute("data-is-restart-required");
+
+  document.addEventListener("DOMContentLoaded", function () {
+    document
+      .querySelectorAll(".plugin-manager-toggle-switch")
+      .forEach(function (toggle) {
+        toggle.addEventListener("click", flip);
+      });
+  });
+
+  function flip(event) {
+    let btn = event.target;
+
+    // trigger
+    fetch(
+      btn.getAttribute("url") +
+        "/make" +
+        (btn.checked ? "Enabled" : "Disabled"),
+      {
+        method: "post",
+        headers: crumb.wrap({}),
+      }
+    ).then((rsp) => {
+      if (!rsp.ok) {
+        rsp.text().then((responseText) => {
+          document.getElementById("needRestart").innerHTML = responseText;
+        });
+      }
+      updateMsg(btn);
```

Comment:
I can't see any particular reason to pass `btn` to `updateMsg`. `updateMsg` is not declared to accept an argument, nor does it use any arguments.

```suggestion
      updateMsg();
```

Reply:
Whoops, this is actually a development leftover. Will remove, thanks for pointing this out.